tools/mxl-gst/CMakeLists.txt

```diff
@@ -23,7 +23,7 @@ if (gstreamer_FOUND)
     endif ()
 
 
-    add_executable(mxl-gst-videotestsrc)
+    add_executable(mxl-gst-videotestsrc videotestsrc.cpp)
```

Comment:
Could you please keep the previous separation of the declaration of the target and its sources?